Docs/Them Lit FYP Information.md

```diff
@@ -52,7 +59,7 @@ The project aims to make exercising fun with VR and Sensors. Body movements will
 
 
 **Fitness involvement:**
-- Requires a lot of movement. Kicking(TBC) and smacking balls are allowed, hence it can be a full body workout.
+- Requires a lot of movement. ~~Kicking(TBC)~~ and smacking balls are allowed, hence it can be a full body workout.
```

Comment:
```suggestion
- Requires a lot of movement. ~~Kicking(TBC) and~~ smacking balls are allowed, and movement is required to move the goals to avoid getting scored on, hence it can be a full body workout.
```